.github/workflows/build-python.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cibw-identifier: ["cp38-manylinux_x86_64", "cp39-manylinux_x86_64", "cp310-manylinux_x86_64", "cp311-manylinux_x86_64"]
```

Comment:
Good-bye Python 3.8 support ....

Reply:
yep.  numpy 1.25 stopped supporting python 3.8, so we're going to too - at least here.

once we get a conda recipe together we can set it up to build for python3.8 and all the different versions of numpy as well, but that's a task for another day